website/docs/docs/get-started-dbt.md

```diff
@@ -85,10 +85,10 @@ Learn more about [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-feature
     icon="dbt-bit"/>
 
 <Card
-    title="dbt Core using GitHub Codespace"
-    body="Learn how to create a codespace and execute the dbt build command."
-    link="https://docs.getdbt.com/guides/codespace?step=1"
-    icon="dbt-bit"/>
+    title="Quickstart for dbt Core using DuckDB"
+    body="Learn how to connect to DuckDB."
+    link="https://docs.getdbt.com/guides/duckdb?step=1"
+    icon="duckdb"/>
```

Comment:
just a heads up the duckdb icon isn't appear here:
<img width="420" alt="Screenshot 2024-12-23 at 12 39 35" src="https://github.com/user-attachments/assets/aaaf8f4d-0fb4-43b4-b83a-ee7efd55d171" />

Reply:
is the icon name correct? or is it `duckdb-seeklogo` like in the guides?

Reply:
Hiya @mirnawong1 

Thanks for checking this out. I saved the file under: duckdb-seeklogo.svg in the light and dark sections of vscode.

Kind Regards
Natalie